test/functional/tests/io_class/test_io_classification.py

```diff
@@ -444,13 +445,15 @@ def test_ioclass_id_as_condition(filesystem):
             ioclass_config_path=ioclass_config_path,
         )
         casadm.load_io_classes(cache_id=cache.cache_id, file=ioclass_config_path)
+        time.sleep(1)
```

Comment:
I think that it might be beneficial to put a comment in here why test sleeps

Reply:
also instead of an arbitrary value we have `MAX_CLASSIFICATION_DELAY = timedelta(seconds=6)` in ioclass_config.py